.github/workflows/conformance.yml

```diff
@@ -109,12 +109,13 @@ jobs:
         working-directory: "gateway-api/conformance"
         run: go test -v -timeout 10m ./ --gateway-class consul-api-gateway
 
-      # TODO Waiting for webhook approval
-      # - name: Report Status
-      #   if: always()
-      #   uses: ravsamhq/notify-slack-action@v1
-      #   with:
-      #     status: ${{ job.status }}
-      #     notify_when: 'failure'
-      #   env:
-      #     SLACK_WEBHOOK_URL: ${{ secrets.ACTION_MONITORING_SLACK }}
+      - name: Report Status
+        if: always()
```

Comment:
Thought on only notifying when the trigger was a scheduled run / when the branch is `main`?

I've had PRs where I ran conformance over and over as I tweek things, and I'm always looking directly at the PR status. Wouldn't wanna spam our channel in Slack.

Reply:
I'd agree with that, I'll get that added.